modin/engines/ray/generic/io.py

```diff
@@ -180,7 +177,7 @@ def read_parquet(cls, path, engine, columns, **kwargs):
         # ensure that when we do the math for the blocks, the partition column
         # will be read in along with a non partition column.
         if columns and directory and any(col in partitioned_columns for col in columns):
-            # partitioned_columns = [col for col in columns if col in partitioned_columns]
+            partitioned_columns = [col for col in columns if col in partitioned_columns]
```

Comment:
Won't this make it so that if we want columns 1 and 5 with the dataset partitioned on 1, 2, and 5, we will get 1 and 5 while pandas returns 1, 2, and 5?